ompi/mca/osc/pt2pt/osc_pt2pt_frag.h

```diff
@@ -172,6 +172,10 @@ static inline int ompi_osc_pt2pt_frag_alloc (ompi_osc_pt2pt_module_t *module, in
 {
     int ret;
 
+    if (request_len > mca_osc_pt2pt_component.buffer_size) {
+        return OMPI_ERR_OUT_OF_RESOURCE;
+    }
+
     do {
         ret = ompi_osc_pt2pt_frag_alloc (module, target, request_len , buffer, ptr, long_send, buffered);
```

Comment:
Ah, ok.  I thought somehow it didn't matter over here.

Will you add c72fb30 to this PR?